python/test/Ice/asyncio/Server.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+#
+# Copyright (c) ZeroC, Inc. All rights reserved.
+#
+
+import Ice
+import asyncio
+from TestHelper import TestHelper
+TestHelper.loadSlice("Test.ice")
+import TestI
+
+
+class Server(TestHelper):
+
+    def run(self, args):
+
+        async def runAsync():
+            loop = asyncio.get_event_loop()
```

Comment:
Use get_running_loop()? (requires 3.7)

Reply:
I'd rather keep get_event_loop if we still test with Python 3.5 on Linux platforms.

Reply:
Ok.